### ISSUE TITLE
verified devices in user info security are buttons which don't do anything

### ISSUE THREAD

**Issue by t3chguy** [vector-im/riot-web#13242]

**Comment by bwindels** [comment]
I'm not sure I understand ... a verified message in `UserInfo` is a button that doesn't work?

**Comment by t3chguy** [comment]
![image](https://user-images.githubusercontent.com/2403652/79731816-b93d9380-82ea-11ea-978b-6b7b03f60671.png)

These trusted devices are still AccessibleButtons with an onClick which is a nop.

![image](https://user-images.githubusercontent.com/2403652/79731856-c6f31900-82ea-11ea-8d88-83d6cea60f6f.png)

**Comment by bwindels** [comment]
Yeah, the plan is to eventually implement https://www.figma.com/file/wwqBo5oAFfk8XKmtd2YaIs/Cross-signing?node-id=6%3A2951, also including a fingerprint to fix https://github.com/vector-im/riot-web/issues/2735. The the work is sort of tracked in that ticket.

**Comment by t3chguy** [comment]
Sure, in terms of accessibility it is really poor right now though. So it might need to get fixed before the new design impl
